src/renderer/components/SettingsView/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import Paper from '@material-ui/core/Paper'
+import InfoIcon from '@material-ui/icons/Info'
+import { SettingsMenu } from './SettingsMenu'
+import { defineMessages } from 'react-intl'
+import { AboutMapeoMenu } from './AboutMapeo'
+
+const m = defineMessages({
+  aboutMapeo: 'About Mapeo'
+})
+
+const tabs = /** @typedef {const} */ [
+  {
+    tabId: 'AboutMapeo',
+    icon: InfoIcon,
+    label: m.aboutMapeo,
+    subtitle: 'Version and build number'
+  }
+]
+
+export const SettingsView = () => {
+  const [menuItem, setMenuItem] = useState(null)
```

Comment:
Define this `useState` so it is not type `any`

The easiest  way to do this in JSDOC is to set an initial state as a variable using jsdoc:

```ts
/** @type {tabs['tabId'] | null} */
  const initialMenuState = /** {const} */ (null)
  const [menuItem, setMenuItem] = useState(initialMenuState)

```

You will need to [define tabs](https://jsdoc.app/tags-typedef.html) with `@typedef`.